src/components/Header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import HeaderAuth from "./HeaderAuth"
+import HeaderUser from './HeaderUser'
+
+const Header = () => {
+
+    const stateUsInfouserData = useSelector(state => state.userInfo.userData)
+    return (
+        <>
+            {!stateUsInfouserData && // ----- AUTH -------
+                <HeaderAuth />
```

Comment:
LoggedOutHeader or UnAuthorizedHeader
 and in save way below

Reply:
Could be done by ternary

Reply:
Ok)) I'll change it to a ternary expression.